src/convertToObject.js

```diff
@@ -3,16 +3,38 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const one = sourceString.split(';');
+  const two = one
+    .map(char => char.trim())
+    .filter(char => char !== '');
+
+  const three = two.map(char => char.split(':'));
+
+  const threeTrim = three.map(char => {
+    return char.map(item => item.trim());
```

Comment:
Instead of using 'map' to trim the elements of each sub-array, consider using 'map' within the 'split' method on line 23. This will eliminate the need for the 'threeTrim' variable and make the code more concise.